src/gui/CMakeLists.txt

```diff
@@ -630,7 +630,7 @@ SET(QGIS_GUI_HDRS
   attributetable/qgsattributetablemodel.h
   attributetable/qgsattributetableview.h
   attributetable/qgsdualview.h
-  attributetable/qgsfeaturefilterwidget.h
```

Comment:
Can you move this to the private header section?

Reply:
Sure. Done